Model/Game.js

```diff
@@ -267,17 +271,108 @@ class Game {
         trashWidth
       ),
     ];
-    this.player = new Ofeq(
-      screenX,
-      screenY,
-      playerHeight,
-      playerWidth,
-      5,
-      99999,
-      300,
-      "B",
-      "Pluto",
-      "Closed"
-    );
+
+    this.docks = [
+      new Dock(screenX, screenY, screenX / 20, 0, dockHeight, dockWidth),
+      new Dock(
+        screenX,
+        screenY,
+        screenX / 20 + screenX / 5,
+        0,
+        dockHeight,
+        dockWidth
+      ),
+      new Dock(
+        screenX,
+        screenY,
+        screenX / 20 + (2 * screenX) / 5,
+        0,
+        dockHeight,
+        dockWidth
+      ),
+      new Dock(
+        screenX,
+        screenY,
+        screenX / 20 + (3 * screenX) / 5,
+        0,
+        dockHeight,
+        dockWidth
+      ),
+      new Dock(
+        screenX,
+        screenY,
+        screenX / 20 + (4 * screenX) / 5,
+        0,
+        dockHeight,
+        dockWidth
+      ),
+    ];
+    switch (satModel) {
+      case 5:
+        this.player = new Ofeq(
+          screenX,
+          screenY,
+          playerHeight,
+          playerWidth,
+          satModel,
+          99999,
```

Comment:
Use a function for the different player options, that get only whatever changes.